docs/admin/extensible-admission-controllers.md

```diff
@@ -126,6 +126,11 @@ initializers:
           - pods
 ```
 
+Once this `initializerConfiguration` named `example-config` is created, the
```

Comment:
Could you move it to after line 137, so it will read like:

"After you create the `initializerConfiguration`, the system will take a few seconds to honor the new configuration. Then, `"podimage.example.com"` will be appended to the `metadata.initializers.pending` field of newly created pods. You should already have a ready "podimage" initializer controller that handles pods whose `metadata.initializers.pending[0].name="podimage.example.com"`. Otherwise the pods will stuck uninitialized."



BTW, there is another occurrence of `metadata.initializers[0]` earlier in the doc, could you replace it with `metadata.initializers.pending[0].name`?